src/assets/locales/fr.json

```diff
@@ -399,19 +443,28 @@
   "reaction-popover.emoji-label.smile": "Sourire",
   "reaction-popover.emoji-label.wave": "Faire coucou",
   "reaction-popover.title": "Réagir",
+  "revoke-token-modal.cancel": "Annuler",
+  "revoke-token-modal.description1": "Êtes-vous sûr(e) de vouloir revoquer ce jeton ?",
```

Comment:
"Êtes-vous sûr(e) de vouloir révoquer ce jeton ?"

Reply:
I forgot the accent everywhere. I fixed it in https://github.com/mozilla/hubs/pull/5002/commits/b30bebc9c01175da9025f359bc50d6cdf65e2a7f